diff --git a/.gitignore b/.gitignore
index 6d5c44db..17355872 100644
--- a/.gitignore
+++ b/.gitignore
@@ -16,3 +16,4 @@ dist/
 bin/
 .DS_Store
 .env
+rust/.minerva-deps/
diff --git a/rust/Cargo.toml b/rust/Cargo.toml
index 5400d26b..828ad01f 100644
--- a/rust/Cargo.toml
+++ b/rust/Cargo.toml
@@ -18,3 +18,6 @@ gtk = "0.18.1"
 x11-dl = "2.21.0"
 [target.'cfg(target_os = "windows")'.dependencies]
 windows = { version = "0.61.1", features = ["Win32", "Win32_UI", "Win32_UI_WindowsAndMessaging", "Win32_System_Registry", "Win32_System_Diagnostics_Etw"]}
+
+[patch.crates-io]
+wry = { path = ".minerva-deps/wry-0.50.5" }
